uberduck_ml_dev/monitoring/generate.py

```diff
@@ -0,0 +1,19 @@
+# AUTOGENERATED! DO NOT EDIT! File to edit: nbs/monitoring.generate.ipynb (unless otherwise specified).
+
+__all__ = ['MODEL_TYPES']
+
+# Cell
+from ..data_loader import prepare_input_sequence
+
+MODEL_TYPES = ["OD", "OD", "OD", "OD", "OD", "D"]
+
+
+def _get_inference(model, vocoder, texts, speaker_ids, symbol_set, arpabet):
+
+    text_padded, input_lengths = prepare_input_sequence(
+        texts, cpu_run=False, arpabet=arpabet, symbol_set=symbol_set
+    )
+    input_ = text_padded, input_lengths, speaker_ids
+    output = model.inference(input_)
+    audio = vocoder.infer(output[1][:1])
+    return audio
```

Comment:
Can you use this function here? https://github.com/uberduck-ai/uberduck-ml-dev/commit/5e87433b0bda51dc6b0193f6f3422d90e23acff7#diff-f8baa94114a0a737cd41dc5522a4397c66a6dfad3a6ed7c9bce1342c00f8ee66R61-R75